tsconfig.json

```diff
@@ -3,13 +3,16 @@
     "allowSyntheticDefaultImports": true,
     "checkJs": false,
     "allowJs": true,
-    "target": "es6",
-    "module": "es6",
+    "target": "es2019",
+    "module": "esnext",
     "strict": true,
     "noImplicitAny": false,
+    "noUnusedLocals": true,
     "lib": [
-      "es2019"
-    ]
+      "es2019",
+      "dom"
```

Comment:
Is the introduction of `dom` ok considering the use of rdflib in Node?

Reply:
The Response type used in rdflib is the one that Fetch uses. I tried the Express response, but it was not compatible. 

Is there a way to only use the Response type, without requiring the `dom` lib?

Reply:
Not sure =\ Have you been able to test the transpiled files with Node- (e.g. NSS) and web-based (e.g. mashlib) systems? If yes, I'll approve this PR, if no, I'll try to test it out myself later and report back.